src/test/java/com/example/demo/mapper/AnimeMapperTest.java

```diff
@@ -0,0 +1,54 @@
+package com.example.demo.mapper;
+
+import com.example.demo.entity.Anime;
+import com.github.database.rider.core.api.dataset.DataSet;
+import com.github.database.rider.spring.api.DBRider;
+import java.util.List;
+import java.util.Optional;
+import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.jupiter.api.Test;
+import org.mybatis.spring.boot.test.autoconfigure.MybatisTest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+
+@DBRider
+@MybatisTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+public class AnimeMapperTest {
+    @Autowired
+    AnimeMapper animeMapper;
+
+    @Test
+    @DataSet(value = "anime.yml")
+    void すべてのアニメが取得できること(){
+        List<Anime> animeList = animeMapper.findAll();
+        assertThat(animeList)
+                .hasSize(2)
+                .contains(
+                        new Anime(1, "Kill la Kill", "Action"),
+                        new Anime(2, "Fairy Tail", "Adventure")
+                );
+    }
+
+    @Test
+    @DataSet(value = "empty.yml")
+    void アニメが存在しない場合取得できないこと(){
```

Comment:
取得できないだと例外になるというようにもみえます。
```suggestion
    void アニメが存在しない場合に空のListが取得できること(){
```
とするほうが明確かと思います。

Reply:
メソッド名を修正しました。

Reply:
コミットハッシュはこちら 1257821f74e4471e927d6352d496d8b97feb6434 です。